Src/MoneyFox.Core/(Pending)/Common/QueryObjects/PaymentQueryExtensions.cs

```diff
@@ -25,6 +25,14 @@ public static IQueryable<Payment> IsExpense(this IQueryable<Payment> query)
         public static IQueryable<Payment> IsIncome(this IQueryable<Payment> query)
             => query.Where(payment => payment.Type == PaymentType.Income);
 
+        /// <summary>
+        ///     Adds a filter to a query for payments of a specified payment type
+        /// </summary>
+        /// <param name="query">Existing query.</param>
+        /// <returns>Query filtered for the specified payment type.</returns>
+        public static IQueryable<Payment> IsPaymentType(this IQueryable<Payment> query, PaymentType paymentType)
```

Comment:
That would be rather easy testable with a unit test. There are existing ones in `PaymentQueryExtensionTests`

Reply:
Definitely! I'll add a unit test.